<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogFTP</class>
 <widget class="QDialog" name="DialogFTP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Local File</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Remote File</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLocalFiles">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>341</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRemoteFiles">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>30</y>
     <width>371</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QTreeView" name="treeView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>881</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonUpload">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>461</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDownload">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>420</y>
     <width>401</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>881</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
